inlang/source-code/editor/src/pages/@host/@owner/@repository/State.tsx

```diff
@@ -265,18 +265,15 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 				const project = solidAdapter(
 					await loadProject({
 						nodeishFs: newRepo.nodeishFs,
+						repo: newRepo,
```

Comment:
Why is `nodeishFs` still required as a parameter? 

I see substantial risk where the SDK uses a different `fs` than the repo. I don't see a reason why we should not immediately remove `nodeishFs` as a parameter. At least only allow allow either nodesihFs or only repo but not both.

Reply:
@samuelstroschein  because not everything is using lix and even has a repo object that uses the loadproject, things like ide extension would first need to migrate to lix and remove the direct fs usage before we can deprecate the parameter.

Reply:
@janfjohannes can we migrate now? otherwise, ide extension projects won't have an id. 

the migration should be simple. the @inlang/ide-extension passes a repo object instead of an fs object.